web/src/components/AppBar/ApiKeyItem.vue

```diff
@@ -12,35 +13,68 @@
       <v-text-field
         ref="apiKey"
         v-model="apiKey"
-        label="Api Key"
-        :readonly="true"
+        label="API Key"
+        readonly
         append-outer-icon="mdi-content-copy"
-        @click:append-outer="copyApiKey"
+        @click:append-outer="copyToClipboard"
       />
     </v-list-item-content>
   </v-list-item>
 </template>
 
 <script>
-import { mapActions, mapState } from 'vuex';
+import girderRest from '@/rest';
 
 export default {
   name: 'ApiKeyItem',
-  computed: {
-    ...mapState('girder', ['apiKey']),
+  data() {
+    return {
+      apiKey: null,
+    };
   },
   created() {
-    this.fetchApiKey();
+    this.fetch();
   },
   methods: {
-    copyApiKey() {
-      const { input: value } = this.$refs.apiKey.$refs;
-      value.focus();
+    copyToClipboard() {
+      const vTextFieldComponent = this.$refs.apiKey;
+      // v-text-field provides some internal refs that we can use
+      // one is "input", which is the actual <input> DOM element that it uses
+      const inputElement = vTextFieldComponent.$refs.input;
+      inputElement.focus();
       document.execCommand('selectAll');
-      value.select();
+      inputElement.select();
       document.execCommand('copy');
     },
-    ...mapActions('girder', ['fetchApiKey', 'reloadApiKey']),
+    async fetch() {
+      let { data } = await girderRest.get(
+        'api_key', {
+          params: {
+            // eslint-disable-next-line import/no-named-as-default-member
+            userId: girderRest.user._id,
+            limit: 50,
+            sort: 'name',
+            sortdir: 1,
+          },
+        },
+      );
+      let [dandiKey] = data.filter((key) => key.name === 'dandicli');
+      if (!dandiKey) {
+        // create a key
+        let { data } = await girderRest.post('api_key', null, {
```

Comment:
```suggestion
        { data } = await girderRest.post('api_key', null, {
```
This line is throwing a linting error because `data` is already declared. I think if you just remove the `let` you can reassign it safely.